text/0000-export-report.md

```diff
@@ -0,0 +1,76 @@
+# Meta
+[meta]: #meta
+- Name: Export Report
+- Start Date: (fill in today's date: 2020-04-01)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: "N/A"
+
+# Summary
+[summary]: #summary
+
+The lifecycle export stage should optionally write a machine parseable metadata "report" about the resulting exported image(s).
+
+# Motivation
+[motivation]: #motivation
+
+The logging output of the lifecycle export stage displays the built image's digest or image id. However, this information is not easily available for a platform that needs to utilize the resulting image from a build. 
+
+Without this information, a platform will need to resort to suboptimal mechanisms such as parsing the exported output or reading the most recently tagged image on the registry. 
+
+
+# What it is
+[what-it-is]: #what-it-is
+
+The lifecycle exporter/creator binary will accept an optional flag `report` which provides a path to `report.toml`. When the flag is supplied the export stage will write a toml representation of the built image for platform consumption.
+
+Here is an overview of the suggested schema for report.toml:
+
+```
+[image]
+tags = ["index.docker.io/image/name:latest", "index.docker.io/image/name:other-tag"]
+identifier = "sha256:c8be1b8f4d60d99c281fc2db75e0f56df42a83ad2f0b091621ce19357e19d853"
+```
+
+By representing this in toml, we can extend this information with additional metadata when needed. 
+
+# How it Works
+[how-it-works]: #how-it-works
+
+The export stage will write report.toml after exporting and caching is complete. There is no default path for report.toml as its location will be platform-specific. If the export steps fails to complete no report.toml is expected to be written.
```

Comment:
I'm inclined to think this is generally useful for most platforms. I'm a big fan of setting a default path that could be overrideable instead.

Reply:
I would suggest `./report.toml` (or ./<new-name>.toml) (similar to `./group.toml` and `./analyzed.toml` default paths) which generally resolves to `/layers/report.toml` given that the working dir is usually the layers dir.

We should call out that the default can be changes with an env var or overrode with an exporter flag.

Reply:
Updated to default to the working directory.